### PR TITLE
tcpreplay: remove libnl dependency and update PKG_LICENSE_FILES.

### PR DESCRIPTION
**Pull request by commodo**
Maintainer: me / @commodo
Compile tested: LEDE trunk ( https://github.com/lede-project/source/commit/18eebfe4e5756bc33b8f04c73477bdb1c1fcb7ee )

Description:
If libnl is enabled, tcpreplay would pick it up (via configure script), and compile with it.
However on link it would fail (at build-time).
So, the simplest fix is to conditionally depend on `libnl-genl`.

Also, updated PKG_LICENSE_FILES.

Signed-off-by: Alexandru Ardelean <ardeleanalex@gmail.com>

### CODE REVIEW

**Comment by champtar** [comment]
@commodo if you don't need libnl, why not disable it (give proper params to configure or patch configure)?

**Comment by commodo** [comment]
Hmmm.
That's also an idea.
Works for me too.
I have no strong preference here.

Will re-spin.

**Comment by jow-** [comment]
In doubt, I'd say always opt for the smaller footprint solution.

**Comment by jow-** [comment]
You probably need something like

    CONFIGURE_VARS += \
      ac_cv_lib_nl_genl_3_genl_connect=no \
      ac_cv_lib_nl_3_nl_cache_alloc=no

**Comment by commodo** [comment]
yep ; that worked; 
@jow thanks for the info ; i've learned something new
decided to also explicitly disable libdbus-1 since it looks like it also may pull that in (if found)

File: net/tcpreplay/Makefile [hunk 1]
```diff
@@ -127,6 +127,10 @@ endef
 CONFIGURE_ARGS += \
 	--enable-force-pf \
 	--enable-dynamic-link \
+	ac_cv_lib_nl_nl_cache_alloc=no \
+	ac_cv_lib_nl_genl_3_genl_connect=no \
+	ac_cv_lib_nl_3_nl_cache_alloc=no \
+	ac_cv_lib_dbus_1_dbus_malloc=no \
 	--prefix="$(PKG_INSTALL_DIR)/usr" \
```
Comment (hnyman) [inline net/tcpreplay/Makefile]: It looks like you have mixed CONFIGURE_ARGS and CONFIGURE_VARS

That might work, but in the long run it is probably best to keep them separately.

**Comment by commodo** [comment]
oh  ;will re-spin & test
my bad

**Comment by commodo** [comment]
re-re-spinned;
@hnyman @thess @jow- 
sorry for the trouble ; thanks for your patience

**Comment by thess** [comment]
Thanks - better solution than mine.

**Pull request merged**